6.3..js

```diff
@@ -0,0 +1,48 @@
+function logFive(sequence) {
+    for (var i = 0; i < 5; i++) {
+        if (!sequence.next()) {
+            break;
+        } else {
+            console.log(sequence.current());
+        }
+    }
+}
+
+function ArraySeq(array) {
+    this.pos = -1;
+    this.array = array;
+}
+
+
+ArraySeq.prototype.next = function() {

```

Comment:
napisałam hasNext i nie wiem czemu nie działa kod, kiedy próbuję wykorzystać to w next. czy coś jest nie tak z kodem, czy nie widzę jakiegoś oczywistego błędu? bo siedziałam nad tym dość długo i nie mogę rozgryźć.